<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2560</width>
    <height>1600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>386</width>
      <height>89</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Macierz A :</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="lineEdit_AROW"/>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string> x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_ACOL"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Calibri</family>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Macierz B :</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLineEdit" name="lineEdit_BROW"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string> x</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_BCOL"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Podaj rozmiar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>lineEdit_AROW</zorder>
   <zorder>lineEdit_ACOL</zorder>
   <zorder>label_2</zorder>
   <zorder></zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2560</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionDodaj"/>
    <addaction name="actionOdejmij"/>
    <addaction name="actionPomn"/>
    <addaction name="actionLosuj"/>
    <addaction name="actionWyznacznik"/>
    <addaction name="actionRz_d"/>
    <addaction name="actionMacierz_tr_jk_tna"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDodaj"/>
   <addaction name="actionOdejmij"/>
   <addaction name="actionPomn"/>
   <addaction name="actionLosuj"/>
   <addaction name="actionWyznacznik"/>
   <addaction name="actionRz_d"/>
   <addaction name="actionMacierz_tr_jk_tna"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDodaj">
   <property name="icon">
    <iconset resource="ico/ikony.qrc">
     <normaloff>:/new/ico/add.png</normaloff>:/new/ico/add.png</iconset>
   </property>
   <property name="text">
    <string>Dodaj</string>
   </property>
  </action>
  <action name="actionOdejmij">
   <property name="icon">
    <iconset resource="ico/ikony.qrc">
     <normaloff>:/new/ico/minus.png</normaloff>:/new/ico/minus.png</iconset>
   </property>
   <property name="text">
    <string>Odejmij</string>
   </property>
  </action>
  <action name="actionPomn">
   <property name="icon">
    <iconset resource="ico/ikony.qrc">
     <normaloff>:/new/ico/razy.png</normaloff>:/new/ico/razy.png</iconset>
   </property>
   <property name="text">
    <string>Pomnóż</string>
   </property>
  </action>
  <action name="actionLosuj">
   <property name="icon">
    <iconset resource="ico/ikony.qrc">
     <normaloff>:/new/ico/losuj.png</normaloff>:/new/ico/losuj.png</iconset>
   </property>
   <property name="text">
    <string>Losuj</string>
   </property>
  </action>
  <action name="actionWyznacznik">
   <property name="icon">
    <iconset resource="ico/ikony.qrc">
     <normaloff>:/new/ico/det.png</normaloff>:/new/ico/det.png</iconset>
   </property>
   <property name="text">
    <string>Wyznacznik</string>
   </property>
  </action>
  <action name="actionRz_d">
   <property name="icon">
    <iconset resource="ico/ikony.qrc">
     <normaloff>:/new/ico/rank.png</normaloff>:/new/ico/rank.png</iconset>
   </property>
   <property name="text">
    <string>Rząd</string>
   </property>
  </action>
  <action name="actionMacierz_tr_jk_tna">
   <property name="icon">
    <iconset resource="ico/ikony.qrc">
     <normaloff>:/new/ico/gauss.png</normaloff>:/new/ico/gauss.png</iconset>
   </property>
   <property name="text">
    <string>Macierz trójkątna</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ico/ikony.qrc"/>
 </resources>
 <connections/>
</ui>
